packages/simcore-sdk/tests/integration/test_node_data_data_manager_.py

```diff
@@ -25,17 +26,17 @@
 # UTILS
 
 
-def _remove_file_or_folder(file_or_folder: Path) -> None:
-    if file_or_folder.is_file():
-        file_or_folder.unlink()
-        assert file_or_folder.exists() is False
-        file_or_folder.touch()
-        assert file_or_folder.exists() is True
+def _remove_path(path: Path) -> None:

```

Comment:
weird naming. it actually re-creates the file

Reply:
tryied soemthing a bit better